packages/react-core/src/components/AlertGroup/examples/AlertGroup.md

```diff
@@ -6,37 +6,68 @@ propComponents: ['Alert', 'AlertGroup', 'AlertActionCloseButton', 'AlertActionLi
 ---
 
 ## Examples
+
+### Variant examples
+
+Alert groups utilize the same variants that are available to [alert components](components/alert/). You can add multiple alert variants to each group.
```

Comment:
I think this could be rephrased a bit, as alert group variants are a bit different than alert variants if I'm understanding the intent of "variant" correctly. Would we consider the alert group variants to be "static", "dynamic" (non toast, which is currently seen as live region examples in the Alert PR), and "toast"?